sdcm/sct_config.py

```diff
@@ -1638,6 +1639,21 @@ class SCTConfiguration(dict):
              help="""reserves instances capacity for whole duration of the test run (AWS only).
              Fallbacks to next availabilit zone if capacity is not available"""),
 
+        dict(name="use_dedicated_host", env="SCT_USE_DEDICATED_HOST", type=boolean,
+             help="""Allocates dedicated hosts for the instances for the entire duration of the test run (AWS only)"""),
+
+        dict(name="aws_dedicated_host_ids", env="SCT_AWS_DEDICATED_HOST_IDS", type=str_or_list_or_eval,
+             help="""list of host ids to use, relevant only if `use_dedicated_host: true` (AWS only)"""),
+
+        dict(name="post_behavior_dedicated_host", env="SCT_POST_BEHAVIOR_DEDICATED_HOST", type=str,
```

Comment:
Can reuse dedicated hosts for different tests? If not, why not to reuse post_behavior of db nodes?
If so, worth to add bit of documentation about it and how to do that.

Reply:
we do want this ability to run multiple tests on the same host, while still clearing the nodes out of it.
hence why needed a separate configuration then DBs